src/interceptors/index.js

```diff
@@ -0,0 +1,16 @@
+import * as xmlHttpRequest from './xmlhttprequest';
+import * as fetch from './fetch';
+
+const interceptor = ({name, reference, fakeService}) => ({
+  assign(routes) {
```

Comment:
Good point, since router assigns them, interceptors themselves are just being enabled and disabled. 👍